kvbc/include/sparse_merkle/internal_node.h

```diff
@@ -166,6 +166,11 @@ std::ostream& operator<<(std::ostream& s, std::optional<T> const& v) {
 class BatchedInternalNode {
  public:
   static constexpr size_t MAX_CHILDREN = 31;
```

Comment:
Isn't the radix 16?

Reply:
We store 16 children at the lowest level of the record then 8, 4, 2 and 1 in the layers above - in total 31 children.